app/views/admin/data_update_scripts/index.html.erb

```diff
@@ -7,23 +7,66 @@
         <th scope="col" class="whitespace-nowrap">Status</th>
         <th scope="col">Created at</th>
         <th scope="col">Run at</th>
+        <th scope="col">Re-run</th>
       </tr>
     </thead>
     <tbody class="crayons-card">
       <% @data_update_scripts.each do |script| %>
-        <tr>
-          <td class="whitespace-nowrap"><%= script.id %></td>
-          <td><%= script.file_name %></td>
-          <td class="justify-content-center">
+        <tr class="<%= 'alert-danger' if script.status == 'failed' %>" id="data_update_script_<%= script.id %>_row">
+          <td id="data_update_script_<%= script.id %>" class="whitespace-nowrap"><%= script.id %></td>
+          <td id="data_update_script_<%= script.id %>_file_name"><%= script.file_name %></td>
+          <td id="data_update_script_<%= script.id %>_status" class="justify-content-center">
             <%= script.status %>
             <% if script.error.present? %>
               <div class="fs-xs"><%= script.error %></div>
             <% end %>
           </td>
-          <td><%= script.created_at %></td>
-          <td><%= script.run_at %></td>
+          <td id="data_update_script_<%= script.id %>_created_at"><%= script.created_at %></td>
+          <td id="data_update_script_<%= script.id %>_run_at" class="whitespace-nowrap"><%= script.run_at %></td>
+          <td id="data_update_script_<%= script.id %>_button" class="whitespace-nowrap">
+            <button onclick="forceRun(<%= script.id %>);return false;" type="button" className="crayons-btn crayons-btn--secondary">
+              Re-run
+            </button>
+          </td>
         </tr>
       <% end %>
     </tbody>
   </table>
 </main>
+
+<script>
+  function forceRun(id) {
```

Comment:
can we move this into a Stimulus controller? I think inline JS should be avoided as much as possible as it also makes modularization difficult and it's ignored by ESLint as far as i know